src/components/modals/ChallengeModal.tsx

```diff
@@ -49,6 +50,8 @@ export async function retryOnChallenge<T, C>(opts: {
 
 export const ChallengeModal = (props: Props) => {
   const { bridge, challengeUri } = props
+  const { context } = useImports()
+  const { isDesktop } = context
```

Comment:
This commit is using something that's not defined yet. Can we move it later in the PR (or just drop it as per the other comment)?

Reply:
Dropped the commit